packages/workbox-build/src/templates/sw-template.js

```diff
@@ -19,6 +19,8 @@ module.exports = `/**
  * See https://goo.gl/2aRDsh
  */
 
+<% if (navigationPreload) { %>workbox.navigationPreload.enable();<% } %>
```

Comment:
I wasn't entirely sure where this should go in the template order-wise. From the docs, it looked like it should go ahead of strategies and routing config.

Reply:
Under the hood, this just installs an `activate` event listener, which won't run until that lifecycle event is fired. So it doesn't matter too much where the code is located.